ibm/service/power/ibm_pi_constants.go

```diff
@@ -33,6 +34,7 @@ const (
 	Arg_SharedProcessorPoolPlacementGroupID = "pi_shared_processor_pool_placement_group_id"
 	Arg_SharedProcessorPoolReservedCores    = "pi_shared_processor_pool_reserved_cores"
 	Arg_SnapshotID                          = "pi_snapshot_id"
+	Arg_SnapShotName                        = "pi_snap_shot_name"
```

Comment:
In the docs, the argument is mentioned as `pi_snapshot_name`. Could you please correct it? 

Also, note in the id we use `snapshot` and in the name `snap_shot`. Hope we can correct this everywhere.

Reply:
yea that's super awkward I'll try to fix the documentation